docker-entrypoint.sh

```diff
@@ -404,6 +413,8 @@ if [ "$haveSslConfig" ] && [[ "$1" == rabbitmq* ]] && [ ! -f "$combinedSsl" ]; t
 		cat "$RABBITMQ_SSL_KEYFILE"
 	} > "$combinedSsl"
 	chmod 0400 "$combinedSsl"
+
+	echo >&2 "WARNING: relying on 'docker-entrypoint.sh' generating a combined SSL file, which will no longer happen in a future release! (https://github.com/docker-library/rabbitmq/pull/424)"
```

Comment:
I'd make this say "combined PEM certificate file" (I wasn't sure what the "combined SSL file" is)